kong/clustering.lua

```diff
@@ -454,25 +455,42 @@ function _M.init_worker(conf)
     assert(shdict, "kong_clustering shdict missing")
 
     -- ROLE = "control_plane"
+    local db = kong.db
 
     kong.worker_events.register(function(data)
       -- we have to re-broadcast event using `post` because the dao
       -- events were sent using `post_local` which means not all workers
       -- can receive it
+
+      -- send event to other workers in the same node
       local res, err = kong.worker_events.post("clustering", "push_config")
       if not res then
         ngx_log(ngx_ERR, "unable to broadcast event: " .. err)
       end
+
+      -- send event to other nodes in the cluster
+      local cache_key = db[data.schema.name]:cache_key(data.entity)
+      local key = fmt("%s:%s", data.operation, cache_key)
+      res, err = kong.cluster_events:broadcast("clustering:push_config", key)
```

Comment:
can’t we just subcribe the invalidations?

Reply:
Talked about this with @bungle privately. I misunderstood how certain things work in the `cluster_events` library. Will redo the PR accordingly.